src/index.js

```diff
@@ -280,17 +286,16 @@ class Libp2p extends EventEmitter {
     )
   }
 
-  // TODO: Update ping
-  // /**
-  //  * Pings the provided peer
-  //  *
-  //  * @param {PeerInfo|PeerId|Multiaddr|string} peer The peer to ping
-  //  * @returns {Promise<Ping>}
-  //  */
-  // ping (peer) {
-  //   const peerInfo = await getPeerInfoRemote(peer, this)
-  //   return new Ping(this._switch, peerInfo)
-  // }
+  /**
+   * Pings the given peer
+   * @param {PeerInfo|PeerId|Multiaddr|string} peer The peer to ping
+   * @returns {Promise<number>}
+   */
+  async ping (peer) {
+    const peerInfo = await getPeerInfoRemote(peer, this)
```

Comment:
I think we should just use `getPeerInfo` for this, as ping really shouldn't support peers we don't know about locally. `getPeerInfoRemote` will search the network if it can't find it locally.